### PR TITLE
fix: removed typos and unnecessary info

### PR DESCRIPTION
**Pull request by RinorG**

### CODE REVIEW

File: src/markdown/jobs/brand-and-content-marketing-manager.mdx [hunk 1]
```diff
@@ -56,8 +56,7 @@ As a Brand & Content Marketing Manager at Ory, you will be responsible for gener
 - At Ory you will be actively involved in shaping and making decisions that will have a significant impact on the company.
 - Access to one of the largest open source communities on the internet of world-class developers and world-changing projects.
 - #PayItForward: As a startup with open source roots, providing access to software is important to us. Ory employees can work 10% of their working hours on open source projects of their choice.
-- A attractive salary and the possibility of employee stock options.
-- Top hardware including two monitors, standing desk, gigabit internet and OS of your choice.
+- An attractive salary and the possibility of employee stock options.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/brand-and-content-marketing-manager.mdx:59]:
```suggestion
- An attractive salary and top hardware, gigabit internet and OS of your choice.
```

Comment (BlueAcidFrog) [inline src/markdown/jobs/brand-and-content-marketing-manager.mdx]:
```suggestion
- An attractive salary and top hardware, gigabit internet and OS of your choice.
```

File: src/markdown/jobs/data-scientist.mdx [hunk 2]
```diff
@@ -52,8 +52,7 @@ As a Data Scientist at Ory, your task will be to evaluate the user journey and p
 - At Ory you will be actively involved in shaping and making decisions that will have a significant impact on the company.
 - Access to one of the largest open source communities on the internet of world-class developers and world-changing projects.
 - #PayItForward: As a startup with open source roots, providing access to software is important to us. Ory employees can work 10% of their working hours on open source projects of their choice.
-- A attractive salary and the possibility of employee stock options.
-- Top hardware including two monitors, standing desk, gigabit internet and OS of your choice.
+- An attractive salary and the possibility of employee stock options.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/data-scientist.mdx:55]:
```suggestion
- An attractive salary and top hardware, gigabit internet and OS of your choice.
```

File: src/markdown/jobs/senior-performance-marketing-manager.mdx [hunk 3]
```diff
@@ -55,8 +55,7 @@ As a senior performance marketing manager at Ory, you will be responsible for ge
 - At Ory you will be actively involved in shaping and making decisions that will have a significant impact on the company.
 - Access to one of the largest open source communities on the internet of world-class developers and world-changing projects.
 - #PayItForward: As a startup with open source roots, providing access to software is important to us. Ory employees can work 10% of their working hours on open source projects of their choice.
-- A attractive salary and the possibility of employee stock options.
-- Top hardware including two monitors, standing desk, gigabit internet and OS of your choice.
+- An attractive salary and the possibility of employee stock options.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/senior-performance-marketing-manager.mdx:58]:
```suggestion
- An attractive salary and top hardware, gigabit internet and OS of your choice.
```

**Review by BlueAcidFrog — APPROVED** [review]

**Comment by vinckr** [comment]
Can you resolve the conflicts @BlueAcidFrog ?

Basically decide which version to keep.

**Comment by vinckr** [comment]
I resolved the conflicts for you @BlueAcidFrog, please review and then merge.

**Review by BlueAcidFrog — APPROVED** [review]
LGTM

**Pull request merged**